iOS/Flows/Conversation/_New/ConversationCollectionViewLayout.swift

```diff
@@ -20,23 +22,28 @@ class new_ConversationCollectionViewLayout: UICollectionViewFlowLayout {
         fatalError("init(coder:) has not been implemented")
     }
 
+    // IMPORTANT: This allows us to layout the cells right to left.
+    override var flipsHorizontallyInOppositeLayoutDirection: Bool {
+        return true
+    }
```

Comment:
Setting this to true allows the layout to respect the right to left content attribute above.

Reply:
Can you comment the property name and class name that this reflects.